youtube_dl/extractor/playerglobewien.py

```diff
@@ -0,0 +1,93 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+import re
+from .common import InfoExtractor
+
+
+class PlayerGlobeWienIE(InfoExtractor):
+    _VALID_URL = r'https?://player.(?:globe.wien|hader.at)/(?:globe-wien|hader)/(?P<id>.*)'
```

Comment:
maybe you could restrict group `id` to exclude query parameters etc.
```python
r'https?://player.(?:globe.wien|hader.at)/(?:globe-wien|hader)/(?P<id>[^/?#]+)
```
example: https://player.hader.at/hader/hader-homestory-teil-6?streamNow=true

Reply:
ok that is a good point, i'll implement it this way, thanks in advance